Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.2.3'
+ruby '2.3.7'
```

Comment:
I upgraded to 2.3.7 because that was what was on my machine when I got it from Fastly. I'm happy to update to 2.5.3 if that is the preference.

Reply:
According to this https://devcenter.heroku.com/articles/ruby-support#supported-runtimes and the error when you try to use the one click button:
```
 !     An error occurred while installing ruby-2.3.7
 !     
 !     This version of Ruby is not available on Heroku-18. The minimum supported version
 !     of Ruby on the Heroku-18 stack can found at:
 !     
 !     https://devcenter.heroku.com/articles/ruby-support#supported-runtimes
```
Ruby 2.3.7 is not supported on the Heroku-18 stack. You can either 
1) try upgrading to a newer ruby and making sure the app still works or
2) setting the app to use the Heroku-16 stack. These [instructions](https://devcenter.heroku.com/articles/upgrading-to-the-latest-stack#upgrading-an-app) may be helpful. I have confirmed that 2.3.7 will deploy to Heroku-16 and that the main functionality of the app works once deployed.